scalding-core/src/main/scala/com/twitter/scalding/serialization/Boxed.scala

```diff
@@ -0,0 +1,470 @@
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed75[K](override val get: K) extends Boxed[K]
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed76[K](override val get: K) extends Boxed[K]
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed77[K](override val get: K) extends Boxed[K]
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed78[K](override val get: K) extends Boxed[K]
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed79[K](override val get: K) extends Boxed[K]
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed80[K](override val get: K) extends Boxed[K]
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed81[K](override val get: K) extends Boxed[K]
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed82[K](override val get: K) extends Boxed[K]
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed83[K](override val get: K) extends Boxed[K]
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed84[K](override val get: K) extends Boxed[K]
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed85[K](override val get: K) extends Boxed[K]
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed86[K](override val get: K) extends Boxed[K]
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed87[K](override val get: K) extends Boxed[K]
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed88[K](override val get: K) extends Boxed[K]
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed89[K](override val get: K) extends Boxed[K]
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed90[K](override val get: K) extends Boxed[K]
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed91[K](override val get: K) extends Boxed[K]
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed92[K](override val get: K) extends Boxed[K]
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed93[K](override val get: K) extends Boxed[K]
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed94[K](override val get: K) extends Boxed[K]
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed95[K](override val get: K) extends Boxed[K]
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed96[K](override val get: K) extends Boxed[K]
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed97[K](override val get: K) extends Boxed[K]
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed98[K](override val get: K) extends Boxed[K]
+
+@DefaultSerializer(classOf[BoxedDefaultSerialization])
+class Boxed99[K](override val get: K) extends Boxed[K]
+
+// TODO this could be any general bijection
+case class BoxedOrderedSerialization[K](box: K => Boxed[K],
```

Comment:
We could do this i guess easily, but maybe it wouldn't be common enough for much of a win? Should probably add an issue around the todo i suppose if we want to do it or just kill the todo?